src/azure-cli-core/setup.py

```diff
@@ -86,16 +92,15 @@
     url='https://github.com/Azure/azure-cli',
     zip_safe=False,
     classifiers=CLASSIFIERS,
-    namespace_packages=[
-        'azure',
-        'azure.cli'
-    ],
     packages=[
+        'azure',
+        'azure.cli',
         'azure.cli.core',
         'azure.cli.core.commands',
         'azure.cli.core.extensions',
         'azure.cli.core.sdk',
         'azure.cli.core.profiles',
     ],
-    install_requires=DEPENDENCIES
+    install_requires=DEPENDENCIES,
```

Comment:
I can't comment upper, but if you want setuptools to work out of the box, you should remove "azure-cli-nspkg" as a direct dependencies of the package. This way the "setuptools" install does not install it, and azure_bdist_wheel will add it automatically to the wheel.